### PR TITLE
[FIX]: removing unneeded NFT module and related code

### PR DESCRIPTION
**Pull request by bengriffin1**
### 📦 Pull Request

For NFT Checkout, approved clients will call our REST API endpoints directly rather than using the SDK. Removing all references to NFT service from this repo.

### CODE REVIEW

File: test/spec/utils/rest/post.spec.ts [hunk 1]
```diff
@@ -28,30 +28,3 @@ test('Successfully POSTs to the given endpoint & stringifies body', async () =>
...
-    {
-      method: 'POST',
-      headers: {
-        'X-Magic-Secret-key': API_KEY,
-        'Content-Type': 'application/json',
-      },
-      body: '{"public_address":"0x0123"}',
-    },
-  ]);
-});
```
Comment (justinnout) [inline test/spec/utils/rest/post.spec.ts:57]: Can we keep this? doesn't look NFT related

Comment (bengriffin1) [reply test/spec/utils/rest/post.spec.ts:57]: I added this in a PR for minting, basically needed to add in the secret key header specifically for minting -- here's where it was added: https://github.com/magiclabs/magic-admin-js/pull/107/files#diff-1dafd08f79b2abad5cee6ab033b9a92fd6e01257c287673b6eb533a9c558335aR32

This test is for this change, which is only needed by minting: https://github.com/magiclabs/magic-admin-js/pull/108/files#diff-3ca6a15d564c2c26d46054f3bde8c43d1e5c79865143e55ad08164353ef58320L54

Comment (justinnout) [reply test/spec/utils/rest/post.spec.ts:57]: Gotcha!

**Review by justinnout — APPROVED** [review]

**Comment by magiclabsFE** [comment]
:rocket: PR was released in [`v1.10.0`](https://github.com/magiclabs/magic-admin-js/releases/tag/v1.10.0) :rocket:

**Pull request merged**